persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageDAOImpl.kt

```diff
@@ -33,8 +34,23 @@ class MessageDAOImpl(private val queries: MessagesQueries) : MessageDAO {
 
     override suspend fun deleteAllMessages() = queries.deleteAllMessages()
 
-    override suspend fun insertMessage(message: MessageEntity) = queries.transaction { insertInDB(message) }
+    override suspend fun insertMessage(
+        message: MessageEntity,
+        isMyMessage: Boolean,
```

Comment:
This is also logic about inserting messages from the self user.  I would still suggest to break it up into a `updateConversationReadDate` and a `updateConversationNotificationDate`.